servers/lib/DEVELOPER.md

```diff
@@ -43,6 +43,25 @@ The URL endpoint for this microservice is located at: `localhost:PORT/lib`
 
 The [API](./API.md) page shows sample queries and responses.
 
+## Manual testing
+
+As of now, some features require manual testing or attention.
+
+### CloudCMD
+
+Due to how CloudCMD works,
+it is required to manually test with `LOCAL_PATH` as an absolute path
+e.g. `C:\DTaaS\files` and as a relative path e.g. `..\..\files`.
+Use the paths relavant to your OS.
```

Comment:
Perhaps provide an example for different OS? With a table 

| OS |Absolute Path |Relative Path|
|:---|:---|:---|
| Windows | |
| Linux | |

With the table, the text could be simplified to,

```txt
Due to how CloudCMD works,
it is required to manually test with `LOCAL_PATH` both as an absolute path and local path
```

Reply:
Prasad will look into it